docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/vm.md

```diff
@@ -51,8 +51,7 @@ sudo firewall-cmd --zone=public --add-port=9880/tcp
 sudo firewall-cmd --zone=public --add-port=9874-9879/tcp
 ```
 
-
-## Create mount paths on the nodes
+## Mount paths on the nodes
```

Comment:
Seems like the solution, but what was the problem

Reply:
Right. I think I fixed it, I'll get dev to review to make sure it's correct.